### ISSUE TITLE
HTML5 @border must not use "0" (zero) value

### ISSUE THREAD

**Issue by drmacro** [dita-ot/dita-ot#3128]
## Expected Behavior

Tables that have no border should either omit the @border attribute set the value to an empty string.

From the HTML5 spec:

"The border content attribute may be specified on a table element to explicitly indicate that the table element is not being used for layout purposes. If specified, the attribute’s value must either be the empty string or the value "1". The attribute is used by certain user agents as an indication that borders should be drawn around cells of the table."

https://www.w3.org/TR/html52/tabular-data.html#the-table-element

Epubcheck applied to EPUBs reports border="0" as an error, which is how I noticed this.

## Actual Behavior

@border is set to "0" (zero).

## Possible Solution

Use "" rather than "0" for no borders.

## Steps to Reproduce

1. Format a table with @frame "none"
2. Inspect the resulting HTML5 <table> elements to observe @border="0"

## Environment

* DITA-OT version: 2.5.4 through 3.1.3
* Operating system and version:
  _(Linux, macOS, Windows)_
* How did you run DITA-OT?
  _(`dita` command, Ant, `startcmd`, oXygen, other editor, CMS, etc.)_
* Transformation type:
html5

**Comment by jelovirt** [comment]
Can't reproduce this with OT 3.2 code.

**Comment by drmacro** [comment]
I could only find one place where @border is set to "0", which is in reference.xsl if @frame is set to "none" on `<property>`:

```
 <table cellpadding="4" cellspacing="0" border="0" class="simpletable properties simpletablenoborder">
```

But I did verify that the code for CALS tables appears to never set @border directly.

This template in ut-d.xsl sets @border on `<img>`, which it appears HTML5 does not allow at all:

```
  <!-- Set the border attribute on an imagemap; default is to always use border="0" -->
  <xsl:template match="*[contains(@class,' ut-d/imagemap ')]" mode="imagemap-border-attribute">
    <xsl:attribute name="border">0</xsl:attribute>
  </xsl:template>
 ```

**Comment by jelovirt** [comment]
Fixed by #3139
